server/src/graphql/index.ts

```diff
@@ -6,21 +6,26 @@ import Doctor from '../entities/doctor';
 import DoctorResolver from './resolvers/doctor.resolver';
 import Patient from '../entities/patient';
 import Consultation from '../entities/consultation';
+import { Response, Request } from 'express';
 
 type dbConnection = EntityManager<IDatabaseDriver<Connection>>;
 export type CustomContext = {
-  doctorRepo: EntityRepository<Doctor>
-  patientRepo: EntityRepository<Patient>
-  consultationRepo: EntityRepository<Consultation>
+  req: Request;
+  res: Response;
+  doctorRepo: EntityRepository<Doctor>;
+  patientRepo: EntityRepository<Patient>;
+  consultationRepo: EntityRepository<Consultation>;
 }
 
-export const apolloServer = async (em: dbConnection): Promise<ApolloServer> => 
+export const apolloServer = async (em: dbConnection): Promise<ApolloServer> =>
   new ApolloServer({
     schema: await buildSchema({
       resolvers: [PatientResolver, DoctorResolver],
```

Comment:
I might get a conflict here, as I've added the Consultation resolver to line 23.